internal/config/config.go

```diff
@@ -148,7 +149,7 @@ func DefaultConfig() *Config {
 // RepoPath returns the filepath to either the config-file location or initial path provided by the user.
 func (c *Config) RepoPath() string {
 	if c.ConfigFilePath != "" {
-		return c.ConfigFilePath
+		return strings.TrimRight(c.ConfigFilePath, filepath.Base(c.ConfigFilePath))
```

Comment:
Why do we need this? We should also update the docs for this function since they seem inconsistent now.